dfs/snapshot.go

```diff
@@ -299,7 +299,7 @@ func (dfs *DistributedFilesystem) DeleteSnapshots(tenantID string) error {
 	return nil
 }
 
-func (dfs *DistributedFilesystem) restoreServices(svcs []*service.Service) error {
+func (dfs *DistributedFilesystem) restoreServices(tenantID string, svcs []*service.Service) error {
```

Comment:
Possibly. given that we know what tenant we are trying to restore it made sense to me to use that information rather than try to infer it.  One could make a case for asserting that the tenant ID inferred from the snapshot matches the expected